.github/ISSUE_TEMPLATE/bug_report_template.yaml

```diff
@@ -0,0 +1,158 @@
+  - type: dropdown
+    id: contribution
+    attributes:
+      label: Willingness to contribute
+      description: The MLflow Community encourages bug fix contributions. Would you or another member of your organization be willing to contribute a fix for this bug to the MLflow code base?
+      options:
+        - Yes. I can contribute a fix for this bug independently.
+        - Yes. I would be willing to contribute a fix for this bug with guidance from the MLflow community.
+        - No. I cannot contribute a bug fix at this time.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: System information
+      description: |
+        Please fill in this bug report template to ensure a timely and thorough response.
+      value: | 
+        - **Have I written custom code (as opposed to using a stock example script provided in MLflow)**:
+        - **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
+        - **MLflow installed from (source or binary)**:
+        - **MLflow version (run ``mlflow --version``)**:
+        - **Python version**:
+        - **npm version, if running the dev UI**:
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe the problem
+      description: |
+        Describe the problem clearly here. Include descriptions of the expected behavior and the actual behavior.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Tracking information
+      description: |
+        For bugs related to the tracking features (e.g. mlflow should log a run in my database but it doesn't), please insert the following code in your python script / notebook where you encountered the bug and run it:
+        ```python
+        print("MLflow version:", mlflow.__version__)
+        print("Tracking URI:", mlflow.get_tracking_uri())
+        print("Artifact URI:", mlflow.get_artifact_uri())
+        ```
+        Then, make sure the printed out information matches what you expect and paste it (with sensitive information masked) in the box below. If you know the command that was used to launch your tracking server (e.g. mlflow server -h 0.0.0.0 -p 5000), please provide it.
+      placeholder: |
+        ```python
+        # code to reproduce issue
+        ```
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Code to reproduce issue
+      description: |
+        Provide a reproducible test case that is the bare minimum necessary to generate the problem.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Other info / logs
+      description: |
+        Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.
+    validations:
+      required: false
```

Comment:
Can we also add a placeholder in this section for the same reason as the one in the code-to-reproduce-issue section?

````
Traceback (most recent call last):
  File "example.py", line 1, in <module>
    raise Exception("error")
Exception: error
````